src/components/AddLessonAdditionalBoxes/AddLessonAdditionalBoxes.scss

```diff
@@ -0,0 +1,58 @@
+@use "../../styles/variables" as *;
+
+.add-lesson {
+    display: flex;
+    flex-direction: column;
+
```

Comment:
Please give .add-lesson a max width so it is easier to style for whoever is going to create the container.